### PR TITLE
Update `Gameplay differences in osu!(lazer)` with actual behaviour

### PR DESCRIPTION
**Pull request by Ladwick**
Classic mod supports (or at least tries its best to resemble) the stable HP system
Catch grading system is different from the rest of game  

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md [hunk 1]
```diff
@@ -35,13 +35,15 @@ This is mostly due to changes to the judgement system. For example, Geki and Kat
 |  |  |
 | :-- | :-: |
 | Breaks backwards compatibility | Yes |
-| Classic mod revert support | No |
+| Classic mod revert support | Yes |
 | Intentionally changed | Yes |
 | Needs further consideration | Yes |
 
-### All game modes use the same grading system
+### All game modes except osu!catch use the same grading system
```
Comment (Walavouchey) [inline wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:42]: would rather just copy directly from the recent news post

```markdown
In osu!(stable), the accuracy (and judgement) requirements for each [grade](/wiki/Gameplay/Grade) are as follows:

| Grade | osu! / osu!taiko | osu!catch | osu!mania |
| :-: | :-- | :-- | :-- |
| SS | 100% | 100% | 100% |
| S | ≥90% (≤1% MEHs/50s, no misses) | ≥98% | ≥95% |
| A | ≥80% (no misses) or ≥90% | ≥94% | ≥90% |
| B | ≥70% (no misses) or ≥80% | ≥90% | ≥80% |
| C | ≥60% | ≥85% | ≥70% |

Meanwhile, osu!(lazer) has these accuracy cutoffs now:

| Grade | osu! / osu!taiko / osu!mania | osu!catch |
| :-: | :-- | :-- |
| SS | 100% | 100% |
| S | ≥95% | ≥98% |
| A | ≥90% | ≥94% |
| B | ≥80% | ≥90% |
| C | ≥70% | ≥85% |
```

image

Comment (Ladwick) [reply wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:42]: What about this title?

```suggestion
### Grading system is different
```

Comment (Ladwick) [reply wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:42]: Actually the changes broke something with the pipes, will fix later, I'm done for today

Comment (Ladwick) [reply wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:42]: Fixed, waiting for feedback on title change

**Review by Walavouchey — APPROVED** [review]

File: wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:42]: ok as is

**Pull request merged**